### ISSUE TITLE
Need few more details to debug the code for my task

### ISSUE THREAD

**Issue by praveena2j** [GenjiB/LAVISH#24]
Hello, 
      Thanks for sharing the code of your interesting work. I am trying to adapt your work for audio-visual speaker verification. In order to improve my results, I need few clarifications:

1. May I know whether you have used mixup augmentation to get the results produced in the paper ?
2. Can you please share the training logs or loss curves so that I can validate the pattern of my loss curves.

Thanks for your time.

**Comment by GenjiB** [comment]
1. No I didn't use it.  If you want to use it, you can refer to this paper (https://github.com/YuanGongND/cav-mae/blob/master/src/dataloader.py).

2. Just a few epochs for AVE task (AVQA need more).

[output.log](https://github.com/GenjiB/LAVISH/files/15123261/output.log)

**Comment by praveena2j** [comment]
Thanks, if possible can you share the hyper parameters used for AViSH training without latent tokens
